src/features/unified-share-modal/flowTypes.js

```diff
@@ -371,6 +371,8 @@ type BaseUnifiedShareProps = CollaboratorAvatarsTypes &
 
 // Prop types for the Unified Share Modal
 export type USMProps = BaseUnifiedShareProps & {
+    /** A custom action to be invoked instead of default behavior when collaborators avatars are clicked */
+    handleCollaboratorAvatarsClick?: () => void,
```

Comment:
nit: for the prop names of callback functions, we usually use the "on" prefix. "handle" would be used instead when we're defining a function/method

e.g. `onCollaboratorListClick`

Reply:
Thanks for the suggestion, adjusted